src/app/core/components/archive-settings-dialog/archive-settings-dialog.component.ts

```diff
@@ -26,15 +27,19 @@ export class ArchiveSettingsDialogComponent implements OnInit {
   ) { }
 
   public ngOnInit(): void {
-    this.api.tag.getTagsByArchive(this.account.getArchive()).then((response) => {
-      this.tags = response.getTagVOs();
-      this.bindTagsToArchive();
-      this.loadingTags = false;
-    }).catch(() => {
-      setTimeout(() => {
-       this.ngOnInit()
-     }, 1000);
-    })
+    const accessRole = this.account.getArchive().accessRole;
+    this.hasAccess = accessRole === "access.role.owner"; /*|| accessRole === "access.role.manager"*/
+    if (this.hasAccess) {
+      this.api.tag.getTagsByArchive(this.account.getArchive()).then((response) => {
+        this.tags = response.getTagVOs();
+        this.bindTagsToArchive();
+        this.loadingTags = false;
+      }).catch(() => {
+        setTimeout(() => {
+         this.ngOnInit()
+       }, 1000);
```

Comment:
Same comment about Sentry and retry limit / exponential backoff.

Reply:
Wait did I make the same comment on the same chunk of code twice? How did I manage to do that? Sorry!